Sources/AppcuesKit/Presentation/Debugger/ScreenCapturing/UIKitElementTargeting.swift

```diff
@@ -145,11 +145,11 @@ internal extension UIView {
 
     private func getAppcuesSelector(autoTag: String? = nil) -> UIKitElementSelector? {
         return UIKitElementSelector(
-            appcuesID: (self as? AppcuesTargetView)?.appcuesID,
-            accessibilityIdentifier: accessibilityIdentifier,
-            accessibilityLabel: accessibilityLabel,
+            appcuesID: (self as? AppcuesTargetView)?.appcuesID.emptyToNil(),
+            accessibilityIdentifier: accessibilityIdentifier.emptyToNil(),
+            accessibilityLabel: accessibilityLabel.emptyToNil(),
             tag: tag != 0 ? "\(self.tag)" : nil,
-            autoTag: autoTag
+            autoTag: autoTag.emptyToNil()
```

Comment:
This works fine, but I think the right place to do this is in the `UIKitElementSelector.init()`. In case there's ever anything else that's creating a selector object, we'd want the same "no empty strings" rule to apply.

Reply:
sure, good point - i'll move the logic there

Reply:
updated!